examples/large_models/Huggingface_accelerate/llama2/Readme.md

```diff
@@ -0,0 +1,41 @@
+# Loading meta-llama/Llama-2-70b-chat-hf on AWS EC2 g5.24xlarge using accelerate
+
+This document briefs on serving large HG models with limited resource using accelerate. This option can be activated with `low_cpu_mem_usage=True`. The model is first created on the Meta device (with empty weights) and the state dict is then loaded inside it (shard by shard in the case of a sharded checkpoint).
+
+### Step 1: Download model Permission
+
+Follow [this instruction](https://huggingface.co/meta-llama/Llama-2-70b-hf) to get permission
```

Comment:
Suggest switching to chat model instead, so that we can showcase the processing for prompts for the chat model